app/k9mail/src/test/java/com/fsck/k9/account/AccountLoaderTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.fsck.k9.account
+
+import app.k9mail.feature.account.common.domain.entity.AccountState
+import app.k9mail.feature.account.common.domain.entity.AuthorizationState
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
+import com.fsck.k9.Identity
+import com.fsck.k9.mail.AuthType
+import com.fsck.k9.mail.ConnectionSecurity
+import com.fsck.k9.mail.ServerSettings
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import com.fsck.k9.Account as K9Account
+
+class AccountLoaderTest {
+
+    @Test
+    fun `loadAccount() should return null when accountManager returns null`() = runTest {
+        val accountManager = FakeAccountManager()
+        val accountLoader = AccountLoader(accountManager)
+
+        val result = accountLoader.loadAccount("accountUuid")
+
+        assertThat(result).isNull()
+    }
+
+    @Test
+    fun `loadAccount() should return account when present in accountManager`() = runTest {
+        val accounts = mapOf(
+            "accountUuid" to K9Account(
+                uuid = "accountUuid",
+            ).also {
```

Comment:
It's unlikely we'll ever change the constructor to add more parameters. So this could be one line (which in my opinion reads much nicer):

```suggestion
            "accountUuid" to Account("accountUuid").apply {
```